Server/ConcordiaCurriculumManager/Repositories/DossierRepository.cs

```diff
@@ -67,8 +67,14 @@ public async Task<bool> SaveCourseDeletionRequest(CourseDeletionRequest courseDe
     }
 
     public async Task<Dossier?> GetDossierByDossierId(Guid dossierId) => await _dbContext.Dossiers
-        .Select(ObjectSelectors.DossierSelector())
         .Where(d => d.Id == dossierId)
+        .Include(d => d.CourseCreationRequests)

```

Comment:
I think for the 3 requests, you'll have to .ThenInclude the Course/NewCourse field, since they're in the selectors. After you do that, can you also double check that when you navigate to the dossier details page on the frontend, it doesn't crash?

Reply:
I can create, modify, and delete courses (I can also edit/delete those requests) via the API.